pkg/multiTenancyPlugins/quota/quota.go

```diff
@@ -80,11 +80,6 @@ func (quotaImpl *DefaultQuotaImpl) Handle(command string, cluster cluster.Cluste
 	case "containerdelete":

```

Comment:
the check is done in quotaMgmt.go , in function DecreaseQuota.
it checks if such container id exists on the tenants list (if contInfo, ok := tenantQuota.containers[id]; ok{ ),
only swarm containers are in this list so k8 container will not be and quota won't be decreased.